.changeset/eighty-suits-admire.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-scaffolder-node': minor
+---
+
+**BREAKING** Fixed file corruption for non utf-8 data in fetch contents
```

Comment:
This isn't a breaking change right? It's going to just write the buffer as is?

Reply:
Agreed, we overlapped in the review - i made a suggestion and committed it, see if you like it

Reply:
@RedlineTriad if you find all this behavior strange, it's just because there's a release coming up today and i thought that maybe we could get it through asap for that :)

Reply:
I thought I should err on the safe side, because if someone somehow relies on it being converted to utf-8 it might break that behaviour, find with changing it though.